lib/rules/no-v-text.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @author tyankatsu <https://github.com/tyankatsu0105>
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+const utils = require('../utils')
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+/**
+ *
+ * @param {VElement} node
+ * @returns {{node: VAttribute | VDirective, value: string}}
+ */
+const getVText = (node) => {
+  const vText = node.startTag.attributes.find(
+    (attribute) =>
+      attribute.key.type === 'VDirectiveKey' &&
+      attribute.key.name.name === 'text'
+  )
+
+  if (!vText) return
+  if (!vText.value) return
+  if (vText.value.type !== 'VExpressionContainer') return
+  if (!vText.value.expression) return
+  if (vText.value.expression.type !== 'Identifier') return
```

Comment:
Why limit it to an Identifier? I think all expressions should be reported.

Reply:
It's my misunderstood.
I don't know how to use `v-text` very much.
I'm going to fix it.

Reply:
I can use `utils.getDirective`!!!

Reply:
I fixed.